interface/forms/requisition/new.php

```diff
@@ -0,0 +1,363 @@
+                       <?php echo text($npi[0]); ?></br>
+                       <?php echo text($npi[1]); ?></br>
+
+                       </div>
+                   </td>
+                   <td style="vertical-align:top">
+                     <font size="4"><strong><?php print xlt("Responsible Party") ?>:</strong></font></br>
+                      <div class="plist">
+                       <?php echo xlt('Name') ?>:             </br>
+                       <?php echo xlt('Address') ?>:          </br>
+                       <?php echo xlt('City,St,Zip') ?>:      </br>
+                       <?php echo xlt('Relationship') ?>:     </br>
+                       </div>
+                       <div class="pFill"><?php echo "/"; ?></br>
+                       <?php echo "/"; ?></br>
+                       <?php echo "/"; ?></br>
+                       <?php if(!empty($responsibleParty)){echo 'self';}
+                             if(!empty($ins[0]['subscriber_relationship']) && $ins[0]['subscriber_relationship'] == 'child'){echo xlt("Parent");}
+
+                       ?></br>
+
+                       </div>
+                   </td>
+
+
+               </tr>
+                  <tr style="height:125px">
+                   <td style="vertical-align:top; width:400px;">
+                      <font size="4"><strong><?php print xlt("Primary Insurance") ?>:</strong></font></br>
+                      <div class="plist">
+                       <?php echo xlt('Bill Type') ?>:</br>
+                       <?php echo xlt('Payor/Carrier Code') ?>:</br>
+                       <?php echo xlt('Insurance Name') ?>:</br>
+                       <?php echo xlt('Insurance Address') ?>:</br>
+                       <?php echo xlt('City,St,Zip') ?>:</br>
+                       <?php echo xlt('Subscriber/Policy') ?>#:</br>
+                       <?php echo xlt('Group') ?> #:</br>
+                       <?php echo xlt('Physician\'s UPIN') ?>:</br>
+                       <?php echo xlt('Employer') ?>:</br>
+                       <?php echo xlt('Relationship') ?>:</br>
+                      </div>
+                    <div class="pFill">
+                       <?php if(empty($ins[0]['name'])){echo "Patient Bill";}else{echo xlt("Insurance");} ?></br>
+                       <?php echo "/"; ?></br>
+                       <?php echo text($ins[0]['name']); ?></br>
+                       <?php echo text($ins[0]['line1']); ?></br>
+                       <?php echo text($ins[0]['city']) .", ". text($ins[0]['state'])." ".text($ins[0]['zip']); ?></br>
+                       <?php echo text($ins[0]['policy_number']); ?></br>
+                       <?php echo text($ins[0]['group_number']); ?></br>
+                       <?php echo "/"; ?></br>
+                       <?php echo text($ins[0]['subscriber_employer']); ?></br>
+                       <?php echo text($ins[0]['subscriber_relationship']); ?></br>
```

Comment:
This is also like gender above and is based on a list_options. So, to present the title of the option, need to do:
``````
<?php echo text(getListItemTitle('sub_relation', $ins[0]['subscriber_relationship'])); ?></br>
``````

Reply:
Again, pulling data from the table not creating a selectable list.

Reply:
see my comment above